### ISSUE TITLE
Add the ability to whitelist channels from Sponsorblock. 

### ISSUE THREAD

**Issue by Toiletmason** [yuliskov/SmartTube#1388]
Some channels have alright content that people feel the need to add blocked sections that are unnecessary while disabling that type of blocked sections are absolutely necessary on others.

**Comment by Pr0mises** [comment]
I noticed it especially in Linus Tech Tips videos... Theres everything marked

**Comment by Salamat-UK** [comment]
> I noticed it especially in Linus Tech Tips videos... Theres everything marked

Yes, whilst a very useful feature, it does make Linus Tech Tips virtually unwatchable, breaking up the feed far too frequently. The option to turn off, on a ‘per channel’ basis, would be very useful!

**Comment by Fatcakes** [comment]
@Pr0mises You still working on this my guy? It seems like it's becoming more and more necessary. 🤣

**Comment by Pr0mises** [comment]
@Fatcakes Hey, I do not sorry, other stuff came in the way. I'm also unsure when I'll be able to do so.

**Comment by Fatcakes** [comment]
@Pr0mises :(

**Comment by remizik** [comment]
Any update on this ?
I would love to be able to exclude specific channels from SponsorBlock :)

Thanks
